File: class_primer/class_primer__constructor.cs
using static System.Console;

internal class Program
{
    class Employee
    {
        int Num;
        string Name;

        internal Employee( int i, string s )
        {
            Num = i;
            Name = s;
        }

        internal int GetNum()
        {
            return Num;
        }

        internal string GetName()
        {
            return Name;
        }
    }

    static void Main()
    {
        int n = int.Parse( ReadLine() );
        var l = new System.Collections.Generic.List<Employee>( n );

        for( int _ = 0; _ < n; _++ ) {
            string[] s = ReadLine().Split();
            var c = System.StringComparison.Ordinal;

            if( string.Compare( s[0], "make", c ) == 0 ) {
                l.Add( new Employee( int.Parse( s[1] ), s[2] ) );
            } else if( string.Compare( s[0], "getnum", c ) == 0 ) {
                WriteLine( l[int.Parse( s[1] ) - 1].GetNum() );
            } else if( string.Compare( s[0], "getname", c ) == 0 ) {
                WriteLine( l[int.Parse( s[1] ) - 1].GetName() );
            }
        }
        return;
    }
}

File: class_primer/class_primer__constructor.py
class employee:
    def __init__(self, num, name):
        self.num = num
        self.name = name

    def getnum(self):
        return self.num

    def getname(self):
        return self.name

n = int(input())
l = []

for _ in range(n):
    s = input().split()

    if s[0] == "make":
        l.append(employee(int(s[1]), s[2]))
    elif s[0] == "getnum":
        print(employee.getnum(l[int(s[1]) - 1]))
    elif s[0] == "getname":
        print(employee.getname(l[int(s[1]) - 1]))
